api/src/main/java/io/knotx/fragments/api/FragmentResult.java

```diff
@@ -152,6 +170,19 @@ public FragmentOperationFailure getError() {
     return error;
   }
 
+  public boolean isSuccess() {
```

Comment:
Either:
- `isSuccess` and `isError` and `isException` 
or
- `isSuccessful` and `isErroneous` and `isExceptional`